### DISCUSSION TITLE
Table 20, Panel 1

### DISCUSSION THREAD

**Discussion by RafalGSU** [cssi-pi-community/2022-meeting#26] [Panel 1 roundtable (Monday morning)]
How NSF can facilitate growth of diverse population of CI researchers and professionals

**Comment by RafalGSU** [comment]
Developing mechanisms for supplementary grant funding to support competitive salaries for research faculty, and post-award maintenance of open source software and data

**Reply by RafalGSU** [reply]
Another idea from our table: How to foster and encourage people to come to CI and do PhD degrees

**Comment by mchenry** [comment]
The comment about needing to increase the diversity of the candidate pool is very important, and, realizing that this is a very long term goal needs to be appreciated.  As a minority I can state that the only reason I am here, the only reason I realized I could get a Ph.D. was because of the McNair scholars program (https://mcnairscholars.com).  Adopting and expanding such programs might be something to consider.
